README.md

```diff
@@ -5,28 +5,29 @@
 
 ## Quickstart
 
-Run Data explorer with a test dataset:
+Run local Data explorer with a test dataset:
 
 * `docker-compose up --build`
 * Navigate to `localhost:4400`
 
-To use a different dataset:
-
-* For each dataset, there is a directory in `api/dataset_config`, e.g.
-  `api/dataset_config/amp_pd`.
-    * If you used https://github.com/DataBiosphere/data-explorer-indexers to
-      index your dataset, we recommend moving your dataset config directory from
-      that repo to `api/dataset_config/MY_DATASET`. If you need to run
-      more commands in the `data-explorer-indexers` repo, you can simply refer
-      to `api/dataset_config/MY_DATASET` in this repo.
-* Index your data into Elasticsearch. You can use one of the indexers at
-  https://github.com/DataBiosphere/data-explorer-indexers, or any other indexer.
-  https://github.com/DataBiosphere/data-explorer-indexers explains how to
-  set up the config files. Specifically:
-  * There must be a file named `dataset.json` that has a `name` field. This
-    determines the name of the Elasticsearch index.
-  * There must be a file named `facet_fields.csv` with the `readable_field_name`
-    column filled out.
+## Run local Data explorer with a specific dataset
```

Comment:
Can we have two separate sections under this? One for setting up data explorer using data-explorer-indexers, and one for setting up data explorer on top of any existing elasticsearch server?

Reply:
3 out 4 steps are identical, regardless of which indexer you use. Rather than duplicate those 3 steps, I think this way is cleaner. I don't think it's a burden to the user that step 2 has 2 bullets. Let's discuss next time you're in the office. :)

Reply:
That makes sense, we should keep it as is then. I made some additional suggestions aimed at clarifying some of the parts I found confusing